handlers/handlers.go

```diff
@@ -194,10 +195,131 @@ func (s *ServiceSet) serviceHTML(db *mbtiles.DB) handlerFunc {
 	}
 }
 
+type tileCoord struct {
+	z    uint8
+	x, y uint64
+}
+
+// extractTileCoord extracts and returns tile coordinates and an optional
+// extension from a byte slice and returns them. In case the byte slice is too
```

Comment:
s/byte slice/string slice/, of cours, but that will become obsolete with the signature change anyway (cf. below)